packages/fxa-payments-server/src/routes/Subscriptions/SubscriptionItem.tsx

```diff
@@ -55,14 +58,34 @@ export const SubscriptionItem = ({
     );
   }
 
+  if (
+    customerSubscription.cancel_at_period_end === false &&
+    !(subsequentInvoice?.total && subsequentInvoice?.period_start)
+  ) {
+    return (
+      <DialogMessage className="dialog-error" onDismiss={locationReload}>
+        <Localized id="product-plan-not-found">
```

Comment:
Could you please update this and add the new Localized ID's to the `main.ftl`

Reply:
done